__tests__/app.test.js

```diff
@@ -21,8 +22,27 @@ describe("GET /api", () => {
       .get("/api")
       .expect(200)
       .then(({ body: { endpoints } }) => {
-        console.log()
         expect(endpoints).toEqual(endpointsJson);
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of topic objects, each of which should have slug and description properties:", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(topics).toEqual(topics_test);
+      });
```

Comment:
In the case of requiring in your test data and asserting directly against it does indeed work in this instance, but when it comes to SQL assigning ID's to your data, this way of doing it won't work as the ID won't be available in the raw data. How else could you check that the data looks how it is meant to, and that the user is receiving all of the data.